assets/models/system/Mixtral-8x7B-v0-1/spec.yaml

```diff
@@ -20,8 +23,17 @@ tags:
       Standard_ND96asr_v4,
     ]
   inference_supported_envs:
-  - vllm
+    - vllm
+  finetune_compute_allow_list:
+    - Standard_ND40rs_v2
```

Comment:
what is the max sequence length supported for ND40? We can remove the compute if we dont have a validation on that part yet and add it incrementally

Reply:
The max supported is 32k but evaluation component might take time, so leaving it to -1 will pick 512 and user can increase it if necessary.

Reply:
512 is a small value. Let us keep a reasonable value of max sequence length in model specific defaults? 
What is user selects 32k as max sequence length, would it work on ND40?

Reply:
Yes 32 k will work on nd40.

Reply:
Shall I set context length to 2048? Also we know with 512 we got benchmarking on-par with mistral FYI.

Reply:
Setting to 2048 as dicussed.